cmd/release/cmd/tag.go

```diff
@@ -162,12 +162,23 @@ var rancherTagSubCmd = &cobra.Command{
 		ghClient := repository.NewGithub(ctx, rootConfig.Auth.GithubToken)
 
 		opts := &repository.CreateReleaseOpts{
-			Tag:    tag,
-			Repo:   "rancher",
-			Owner:  rancherRelease.RancherRepoOwner,
-			Branch: rancherRelease.ReleaseBranch,
+			Tag:          tag,
+			Repo:         "rancher",
+			Owner:        rancherRelease.RancherRepoOwner,
+			Branch:       rancherRelease.ReleaseBranch,
+			ReleaseNotes: "",
+		}
+		fmt.Printf("creating release options: %+v\n", opts)
+		if dryRun {
```

Comment:
Should you remove `dryRun` from here also? 
@tashima42

Reply:
This is a different dry run, it’s a global flag